src/range.rs

```diff
@@ -863,6 +863,13 @@ pub mod tests {
     proptest! {
 
         // Testing negate ----------------------------------
```

Comment:
This "Testing negate" comment should probably move after the serde round trip test.

Reply:
Good call.